salt/cloud/clouds/vmware.py

```diff
@@ -2968,6 +2971,109 @@ def create(vm_):
                 reloc_spec.host = host_ref
             else:
                 log.error("Specified host: '%s' does not exist", host)
+
+        if instant_clone:
+            instant_clone_spec = vim.vm.InstantCloneSpec()
+            instant_clone_spec.name = vm_name
+            instant_clone_spec.location = reloc_spec
+
+            event_kwargs = vm_.copy()
+            if event_kwargs.get("password"):
```

Comment:
It looks like this is duplicating a lot of code used later in an already very big function. Is it possible to condense the code here without losing functionality?

Reply:
How to make a changelog file and Documentation for the feature?

Reply:
>How to make a changelog file

Here is the documentation on how to add a changelog file: https://docs.saltproject.io/en/latest/topics/development/changelog.html

>Documentation for the feature

The best place to add the documentation for this new option `instant_clone` would be here: doc/topics/cloud/vmware.rst . 

> Let me check if I can condense code without losing functionality.

Thanks :)

Reply:
hey @Ch3LL I was already using relocate spec for instant clone spec and does not use config spec like a simple clone and condensing code would remove instant clone functionality/salt standards for all the specified Parameters for instant clone , hope I do not need to condense the code 😩

Reply:
Thanks for verifying. This should be fine for now then.